ts/components/basic/SessionButton.tsx

```diff
@@ -55,6 +56,7 @@ export const SessionButton = (props: Props) => {
       role="button"
       onClick={onClickFn}
       data-testid={dataTestId}
+      style={{ margin }}
```

Comment:
doing the way you want it makes React regenerate a new unique className. In that component used everywhere else, we should try to avoid this, but we can check that when we get to breaking SessionButton down.

Reply:
Interesting. Noted and I'll be careful.